backend/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "spektate",
+  "version": "0.1.0",
+  "license": "MIT",
+  "dependencies": {
+    "@types/express": "^4.17.0",
+    "spektate": "^1.0.10",
+    "express": "^4.17.1"
+  },
+  "scripts": {
+    "start": "ts-node src/index.ts",
+    "build": "tsc src/** --outDir ../build/backend/ && tsc src/server.ts --outDir ../build/",
+    "build-prod": "tsc src/** --outDir ./build/",
```

Comment:
can you explain why we need to build  and build-prod? thanks

Reply:
`yarn build` is for local builds, and `yarn build-prod` is being used by dockerfile. I don't see a whole lot of use for `yarn build` for local builds though, so i could just remove `yarn build` and rename build-prod to build. `yarn build` is currently being run in pipelines but they can use either one.